docs/_docs/collections.md

```diff
@@ -89,18 +89,59 @@ choice and written out to `<dest>/my_collection/some_subdir/some_doc.html`.
 
 ## Configuring permalinks for collections {#permalinks}
 
-You can customize the [Permalinks](../permalinks/) for your collection's documents by setting `permalink` property in the collection's configuration as follows:
+You can specify a pattern for the URLs where your Collection documents will reside with the [`permalink` property](../permalinks/):
 
 ```yaml
 collections:
-  my_collection:
-    output: true
-    permalink: /awesome/:path/:title.:output_ext
+  categories:
+    output: true                  # without this flag, this Collection's documents will not be generated at build time
+    permalink: /category/:name    # custom permalink definition
 ```
 
-In this example, the collection documents will the have the URL of `awesome` followed by the path to the document and its file extension.
+### Examples
+
+For a collection named `apidocs` with the following source file structure,
+
+```
+├── _apidocs
+│   └── archive
+│       └── deprecated.md
```

Comment:
We *could* use a more generic file name, but I don't mind the current example here, seems pretty explanatory to me.